internal/security/pkiinit/cert/x509Config.go

```diff
@@ -0,0 +1,129 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+//
+// SPDX-License-Identifier: Apache-2.0'
+//
+package cert
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+const (
+	skFileExt   = ".priv.key"
+	certFileExt = ".pem"
+)
+
+// RootCA parameters from JSON: x509_root_ca_parameters
+type RootCA struct {
```

Comment:
These struct types are already declared here:
https://github.com/edgexfoundry/edgex-go/blob/master/internal/pkg/config/x509.go

You may be able to re-use some of the file access code from there as well.

Reply:
@tsconn23 done!